scripts/redirects.js

```diff
@@ -0,0 +1,79 @@
+var fs = require('fs');
+var glob = require('glob');
+var utils = require('../lib/utils');
+
+var MASTER = 'master';
+
+hexo.extend.generator.register('blog-redirects', function () {
+  return expandRedirectObjs([
+    ['blog/2015/12/16/0.0.10-release/', 'blog/introducing-aframe/'],
+    ['blog/2015/12/16/introducing-aframe/', 'blog/introducing-aframe/'],
+    ['blog/2016/03/31/aframe-v0.2.0/', 'blog/aframe-v0.2.0/']
+  ]);
+});
+
+hexo.extend.generator.register('docs-redirects', function () {
+  var redirectObjs = [getDocRootRedirectObjs(), getPreVersionedRedirectObjs()];
+  redirectObjs.push([
+    ['docs/', 'docs/' + hexo.config.aframe_version + '/guide/'],
+    ['docs/guide/', 'docs/' + hexo.config.aframe_version + '/guide/'],
+    // Pre-versioned 0.1.0 removed pages redirects.
+    ['docs/core/templates.html', 'docs/0.1.0/templates.html'],
+    ['docs/guide/entering-vr.html', 'docs/0.1.0/entering-vr.html'],
+    // Pre-versioned 0.1.0 -> 0.2.0 redirects.
+    ['docs/components/loader.html', 'docs/0.2.0/components/collada-model.html'],
+    ['docs/core/animation.html', 'docs/0.2.0/core/animations.html'],
+    ['docs/core/assets.html', 'docs/0.2.0/core/asset-management-system.html'],
+    ['docs/core/mixin.html', 'docs/0.2.0/core/mixins.html'],
+    ['docs/guide/cameras-and-lights.html', 'docs/0.2.0/guide/building-a-basic-scene.html'],
+    ['docs/guide/installation.html', 'docs/0.2.0/guide/getting-started.html'],
+    ['docs/guide/objects.html', 'docs/0.2.0/guide/building-a-basic-scene.html'],
+    ['docs/guide/positioning.html', 'docs/0.2.0/guide/building-a-basic-scene.html#transforming-the-box'],
+    ['docs/primitives/a-cube.html', 'docs/0.2.0/primitives/a-box.html'],
+    ['docs/primitives/a-model.html', 'docs/0.2.0/primitives/a-collada-model.html']
+  ]);
+
+  // Merge.
+  return expandRedirectObjs([].concat.apply([], redirectObjs));
```

Comment:
Updated the comment to be more correct and explain thoroughly that it's to flatten.